sources/ClangSharp.PInvokeGenerator/PInvokeGenerator.VisitDecl.cs

```diff
@@ -1096,7 +1099,7 @@ private void VisitRecordDecl(RecordDecl recordDecl)
                     VisitConstantArrayFieldDecl(recordDecl, constantArray);
                 }
 
-                if (hasVtbl)
+                if (cxxRecordDecl != null && hasVtbl)
```

Comment:
just a note, `hasVtbl` shouldn't ever be `true` if `cxxRecordDecl` is null; so this isn't "strictly" needed.